lib/ansible/plugins/strategy/__init__.py

```diff
@@ -814,7 +814,7 @@ def _load_included_file(self, included_file, iterator, is_handler=False):
 
         except AnsibleError as e:
             if isinstance(e, AnsibleFileNotFound):
-                reason = "Could not find or access '%s' on the Ansible Controller." % to_text(included_file._filename)
+                reason = "Could not find or access '%s' on the Ansible Controller." % to_text(e.file_name)
```

Comment:
we might need both unless file_name is always absolute

Reply:
`file_name` is absolute. It was used in the past, I just made an incorrect assumption when I made this change to override the reason.